public/app/app.js

```diff
@@ -36,7 +36,7 @@
           && $stateParams.id) {
 
           event.preventDefault();
-
+          //this is broken 
```

Comment:
this is the initial comment I left while I first investigated the problem, I will remove the comment now.